concrete/single_pages/dashboard/system/basics/timezone.php

```diff
@@ -1,74 +1,76 @@
 <?php
 defined('C5_EXECUTE') or die('Access Denied.');
 ?>
-<form method="POST" id="user-timezone-form" action="<?= $view->action('update') ?>">
-    <?php $token->output('update_timezone') ?>
+<form method="POST" id="user-timezone-form" action="<?= $view->action('update'); ?>">
+    <?php $token->output('update_timezone'); ?>
```

Comment:
@mnakalay did you add those semicolons manually?

Reply:
@mlocati I don't remember doing it but I'm also pretty sure my IDE doesn't do that on its own so I'm going to guess yes...

I'll remove them ASAP

Reply:
I just checked and I can confirm my IDE did it...

Reply:
Don't use your IDE default settings: use the `c5:phpcs` CLI command.

```sh
./concrete/bin/concrete5 c5:phpcs fix <path> [<path>...]
```

PS: I configured my IDE so that it runs the script above against the currently open file